README.md

```diff
@@ -35,7 +35,18 @@ For more details, see [the quick start](http://gigalixir.readthedocs.io/en/lates
    2. `gigalixir login`
    3. `APP_NAME=$(gigalixir create --name course-planner)`
    4. `gigalixig set_config $APP_NAME DATABASE_URL $YOUR_DB_URL`
+      
+      If you don't have a database, you can create one by running `gigalixir create_database $APP_NAME` and waiting for it to become `AVAILABLE`.
    5. `git push gigalixir master`
-   6. `gigalixir migrate $APP_NAME`
-
-
+   
+   6. `MIX_ENV=prod mix release --env=prod`
+      `MIX_ENV=prod DATABASE_URL="$YOUR_DB_URL" mix ecto.migrate`
+
+      `$YOUR_DB_URL` is in the format: `ecto://user:pass@host:port/db` and you can find those parameters by running `gigalixir databases $APP_NAME`
+
+      This workaround is necessary because the app won't start without running the migrations.
+   
+   
+   7. Copy the contents of the `priv/repo/seeds.exs` file
```

Comment:
are sure this works? seeds is an exs file, the remote console is an erlang console

Reply:
Yes, `gigalixir`'s remote console is `iex` (I believe it connects through a different node but not sure).